marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/endpoints/collector.sjs

```diff
@@ -42,7 +42,13 @@ if (!step) {
 }
 let options = requestParams.options ? JSON.parse(requestParams.options) : {};
 
-let flowDoc= datahub.flow.getFlow(flowName);
+let flowDoc = null;
+if (options.datahubAdHocFlow) {
+  flowDoc = options.datahubAdHocFlow;
+} else {
+  datahub.flow.getFlow(flowName);
```

Comment:
I believe this should be
```suggestion
  flowDoc = datahub.flow.getFlow(flowName);
```

Reply:
Thanks, I just fixed this - this won't get merged yet, just a prototype for now.